eland/tests/ml/test_imported_ml_model_pytest.py

```diff
@@ -228,8 +228,10 @@ def test_random_forest_regressor(self, compress_model_definition):
     @pytest.mark.parametrize("compress_model_definition", [True, False])
     def test_xgb_classifier(self, compress_model_definition):
         # Train model
-        training_data = datasets.make_classification(n_features=5)
-        classifier = XGBClassifier(booster="gbtree")
+        training_data = datasets.make_classification(
```

Comment:
Can we add an additional test (or parametrize this test) so we maintain both the binary and multi classification?

Reply:
@sethmlarson added a new parameter :)